app/frontends/jobs.ts

```diff
@@ -363,13 +363,12 @@ export async function getWorkItemsForWorkflowUI(
       const nextPage = pageLinks.find((l) => l.rel === 'next');
       const previousPage = pageLinks.find((l) => l.rel === 'prev');
       setPagingHeaders(res, pagination);
-      const workflowSteps = await getWorkflowStepsByJobId(db, job.jobID);
       res.render('jobs/workflow-items-table', {
         workItems,
         workflowItemUpdated() { return (new Date(this.updatedAt).toISOString()); },
         workflowItemCreated() { return (new Date(this.createdAt).toISOString()); },
         workflowItemBadge() { return badgeClasses[this.status]; },
-        workflowItemStep() { return workflowSteps[this.workflowStepIndex - 1].serviceID; },
+        workflowItemStep() { return sanitizeImage(this.serviceID); },
```

Comment:
Is `this` the workflow item in this context?

Reply:
Yup!